.github/workflows/issue-management-stale-action.yml

```diff
@@ -34,3 +34,38 @@ jobs:
             as needing author feedback and has not had any activity for 7 days.
             It will be closed automatically if there is no response from the author
             within 7 additional days from this comment.
+
+      - uses: actions/stale@3a9db7e6a41a89f618792c92c0e97cc736e1b13f # v10.0.0
+        with:
+          days-before-stale: 365
```

Comment:
if we wanted to take a more iterative approach, we could probably even start this at like 2 or 3 years, see what it yields, and then lower it until we get it to 365.